### ISSUE TITLE
Pre-populated inputs

### ISSUE THREAD

**Issue by cstarcher** [TexasDigitalLibrary/Vireo#1672]
Relevant Components:

- All Inputs

Description
Best practices suggest that inputs not be populated with filler text on page load. The labels will provide the information necessary, and the reminder text will provide it again. There is no need for the filler text, and it causes two particular challenges.

First, it is redundant, which causes an increase in cognitive load, especially for users who cannot see the element. Currently, the way the element is coded, assistive technology will announce the purpose of the input, then announce that there is an input, then announce the filler text (the purpose of the input again).

Second, it causes more work for every user. Generally, it is expected that a user should navigate to the input and start typing. In this case, something must be done about the text that already exists inside the input.

Solution
Best practice: Take out the filler text.

**Comment by jcreel** [comment]
Looks like our placeholder text is innocuous and not unhelpful.

**Comment by cstarcher** [comment]
After reviewing inputs with a screen reader, my conclusions are as follows:

The first concern of redundancy and increased cognitive load is valid. The inputs are announced correctly when entering the element and then are announced again when reading the placeholder.

The second concern regarding existing text in the input is not valid. There is no text in the input that must be replaced. The visible text is not actual text in the field, and it disappears when the user begins to interact with the input.

It is my opinion that the placeholder text is redundant, provides no benefit for visual or audible interaction, and is a potential hindrance for those using a screen reader.

**Comment by kaladay** [comment]
It looks like we are dealing with Screen-Readers that directly violate the standards.
As per-specification, the screen-reader should only read the placeholder if an aria-label is not present. The screen-reader such as JAWS is introducing this accessibility problem rather than our website.

The documentation that references how a user-agent (such as a screen-reader) should perform does not include the `placeholder` attribute and the HTML5 specification explicitly states that a `placeholder` is not a label. Everything here is pointing to the screen-reader being at fault and introducing the accessibility violation.

Having said that, it would be easier to remove the placeholder to just not deal with this problem (but at the cost of reducing accessibility that the placeholder provides for users who do use it).

Maybe `aria-placeholder` can be used here, instead?

References:
- https://www.w3.org/TR/accname-1.1/#step2
- https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-placeholder
- https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#placeholder
- https://stackoverflow.com/questions/70184492/wcag-screen-reader-nvda-reading-placeholder-information
- https://www.w3.org/WAI/tutorials/forms/labels/#using-aria-label
- https://web.archive.org/web/20230331010336/https://html.spec.whatwg.org/multipage/input.html#the-placeholder-attribute

**Comment by cstarcher** [comment]
Consensus is that the best solution at this time is to remove the placeholders.
